pkg/client/client.go

```diff
@@ -56,43 +57,73 @@ func newUnixSocketClient(ctx context.Context, config *Config) (*Client, error) {
 	}, nil
 }
 
-func newNetworkClient(ctx context.Context, config *Config) (*Client, error) {
+func newNetworkClient(ctx context.Context, config *Config) (*Client, error) {  //problem somewhere here
+
 	certificate, err := tls.LoadX509KeyPair(
-		config.CertFile,
-		config.KeyFile,
+		config.CertFile,	
+		config.KeyFile,		
```

Comment:
```suggestion
		config.CertFile,
		config.KeyFile,
```

Please, fix spacing and formatting issues.